src/views/TabbedView.vue

```diff
@@ -1,14 +1,58 @@
 <template>
-  <div></div>
-</template>
+  <div id="tabbed-page" class="container">
+    <section class="tabs">
+      <tab v-for="tab in tabs" :key="tab.name" :tab="tab" @tab-clicked="getTabItem(tab)" 
+        v-model:activeTab="activeTab"
+      />
+    </section>
+    <section v-if="!loading" class="tab-item">
+      <tab-item :item="pokemon" :columns="pokemonAbilities" v-if="pokemon"/>
+      <div v-else-if="error">
+        <p class="error">{{ error }}</p>
+      </div>
+    </section>
+    <section v-else class="loader-wrapper">
+      <loader-round/>
+    </section>
+  </div>
 
+</template>
 <script>
+import { ref, onBeforeMount, defineAsyncComponent } from "vue"
+import usePokemons from "@/composables/usePokemons"
+import Tab from '@/components/shared/Tab.vue';
+
 export default {
-  name: "TabbedView"
+  name: "TabbedView",
+  components: {
+    Tab,
+    TabItem: defineAsyncComponent(() =>  import('@/components/shared/TabItem.vue')),
+    LoaderRound: defineAsyncComponent(() =>  import('@/components/base/LoaderRound.vue'))
+  },
+  setup() {
+  
+    const tabs = ref([ {name:'charmander', imageSrc: require('@/assets/images/charmander.jpg')} , {name:'pikachu', imageSrc: require('@/assets/images/pikachu.jpg')}, 
+    {name:'ditto', imageSrc: require('@/assets/images/ditto.png')}])
+
+    const activeTab = ref('pikachu')
```

Comment:
If you change pikachu here, will you remember to change it down there?

Reply:
how about this ?
`const activeTab = ref(tabs.value[1].name);`

Reply:
I menant extracting the 'pikachu' string literal to a variable.

Reply:
like an .env variable ?